src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img"></div>
+
+      <h1 class="card__text card__text--head">
```

Comment:
Headings should be used for section titles. If 'APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)' is not a section title, consider using a different tag, like <p> or <div>.